python/fledge/services/core/api/filters.py

```diff
@@ -454,8 +470,8 @@ async def delete_filter_pipeline(request: web.Request) -> web.Response:
                 status_code = resp.status
                 jdoc = await resp.text()
                 if status_code not in range(200, 209):
-                    _LOGGER.error("Error code: %d, reason: %s, details: %s, url: %s", resp.status, resp.reason, jdoc,
-                                  put_url)
+                    _LOGGER.error("Delete {} filter pipeline; Error code: {}, reason: {}, details: {}, url: {}"
```

Comment:
Faile to delete filters pipeline for `<service/task>`. Storage error code: ......